src/routes/app/all-transactions.svelte

```diff
@@ -19,6 +24,54 @@
     if(currentPage > lastPage) goto("/app/all-transactions");
   }
 
+  function roundCurrency (val: number | string): string {
+    if(val === "?") return val;
+    if(typeof val === "string") val = parseFloat(val);
+    return val.toFixed(7);
+  }
+
+  async function getAllTransactions (): Promise<{ id: string, amount: number, status: string }[]> {
+    if(!process.browser) return [];
+
+    // @ts-ignore
+    const client = new Arweave({
+      host: "arweave.net",
+      port: 443,
+      protocol: "https",
+      timeout: 20000,
+    });
+
+    let 
+      query = equals("from", $address),
```

Comment:
In the future, we will only want to query the tx relating to Verto 😄

```suggestion
      query = and(
          // equals("App-Name", "verto"),
          equals("from", $address),
      ),
```

Reply:
I appreciate the feedback, but I think we should display all transactions from a given wallet instead of tx specific to Verto (for this section).

Thanks though!